app/scripts/modules/amazon/src/pipeline/stages/bake/bakeStage.html

```diff
@@ -6,6 +6,14 @@
     <stage-config-field label="Regions">
       <checklist items="regions" model="stage.regions" inline="true" include-select-all-button="true"></checklist>
     </stage-config-field>
+    <stage-config-field label="Skip Region Detection">
```

Comment:
I would add a `help-key`, something to the effect of 

> By default, Spinnaker will detect regions to bake in from downstream deploy stages. To prevent failed deploys from accidentally missed regions during the bake process. This setting will disable this detection mechanism

Reply:
Thanks for your feedback @marchello2000 I agree we can give more context of what the checkbox does